bodhi/server/models.py

```diff
@@ -1154,6 +1146,28 @@ def get_pkg_committers_from_pagure(self):
         # contains FAS group names with commit access.
         return list(committers), list(groups)
 
+    def hascommitaccess(self, username: str, branchname: str) -> bool:
+        """
+        Check on Pagure if a user has commit access on the package/branch.
+
+        Raises:
+            RuntimeError: If Pagure did not give us a 200 code.
+        """
+        pagure_url = config.get('pagure_url')
+        namespace = config.get('pagure_namespaces')[self.type.name]
+        if self.type == ContentType.flatpak:
+            # branchname override for all releases
+            branchname = config.get('pagure_flatpak_main_branch')
+        elif self.type == ContentType.module:
+            # There's no direct map between stream and branch names so just
+            # use main branch
+            branchname = config.get('pagure_module_main_branch')
```

Comment:
Should we document here why we override the default branch for modules and flatpaks and not the others?

Reply:
Isn't the comment above the override enough?

Reply:
aouch, ENEEDGLASSES :(